database/migrations/001-punishments-schema.sql

```diff
@@ -0,0 +1,44 @@
+--------------------------------------------------------------------------------
+-- Up
+--------------------------------------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS punishments (
+	punish_type TEXT NOT NULL,
+	user_id TEXT NOT NULL,
+	ips TEXT NOT NULL,
+	user_ids TEXT NOT NULL,
+	expire_time INTEGER NOT NULL,
+	reason TEXT NOT NULL,
+	PRIMARY KEY (punishType, userid)
+);
+
+CREATE TABLE IF NOT EXISTS room_punishments (
+	punish_type TEXT NOT NULL,
```

Comment:
Is the `_` delimiter standard? I hope this sort of thing is standardized somewhere.

Reply:
Isn't a `_` delimiter standard SQL naming?  I'm not sure what you mean by 'I hope this sort of thing is standardized somewhere' though, standardized where exactly?

Reply:
It might be? I don't know what standard SQL naming actually is.

Reply:
https://stackoverflow.com/questions/7662/database-table-and-column-naming-conventions

StackOverflow seems to want UpperCamelCase, which I don't think I've seen much of.

Reply:
@scheibo?

Reply:
Yeah, I've mostly seen `_`-delimited in SQL databases, so I just assumed that it was the standard.

Reply:
SQLite docs seems to follow that standard (`_`-delimited).

Reply:
https://www.sqlite.org/foreignkeys.html doesnt seem to show `_`, it just does `songartist`?

Reply:
Google internal SQL styleguide favors `UpperCamelCase` for table names and `snake_case` for column names, FWIW.

Reply:
https://www.sqlite.org/cli.html

They use `snake_case` here, it seems like they can't even decide on one standard :p.

Reply:
I think either of these options are fine are:

\- `UpperCamelCase` for tables and `snake_case` for columns 

\- `snake_case` for both.

Reply:
SQLite's docs seems to be using snake_case for table names and nospaces for column names. This sucks.

The docs I've seen that recommend snake_case seem to be for databases that require quoting for identifiers that aren't all-lowercase, which is a good reason but not necessarily relevant to us. :[

Reply:
I'm leaning towards snake_case here. I'll trust JetOU that it's standard.

Reply:
I'm fine with everything being snake case